src/components/WorkPoolEditForm.vue

```diff
@@ -18,8 +18,17 @@
           <p-number-input :id="id" v-model="concurrencyLimit" placeholder="Unlimited" :min="0" />
         </template>
       </p-label>
+
+      <p-label v-if="can.access.workers" label="Type">
+        <WorkPoolTypeSelect v-model:selected="type" disabled />
```

Comment:
```suggestion
        <WorkPoolTypeSelect :selected="type" disabled />
```
Since you don't intend for this value to be updatable by the component, you don't need to v-model this